### PR TITLE
OSDOCS-3186: Creating new machine type table for arm instances

### PR DESCRIPTION
**Pull request by kelbrown20**
https://issues.redhat.com/browse/OSDOCS-3186
Versions 4.10+ 

Preview: https://deploy-preview-40767--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-supported-aws-machine-types_installing-aws-customizations

### CODE REVIEW

File: modules/installation-supported-aws-machine-types.adoc [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 The following Amazon Web Services (AWS) instance types are supported with {product-title}.
 
-.Instance types for machines
+.Instance types for x86_64 machines
```
Comment (kmccarron-rh) [inline modules/installation-supported-aws-machine-types.adoc:16]: One observation - most references I have found are to the x86_64 architecture. Might just want to check that `x86_64 machines` is a sound reference.

File: modules/installation-supported-aws-machine-types.adoc [hunk 2]
```diff
@@ -361,3 +361,87 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
 
 |===
 ====
+
+.Instance types for arm64 machines
```
Comment (kmccarron-rh) [inline modules/installation-supported-aws-machine-types.adoc:365]: Same comment as above - question about referring to a machine by the architecture.

**Review by kmccarron-rh — COMMENTED** [review]
A minor comment regarding machine vs architecture. I'm not so familiar with this area. I'll leave it to you on which way to go with this. Everything else, lgtm!

File: modules/installation-supported-aws-machine-types.adoc [hunk 3]
```diff
@@ -361,3 +361,87 @@ The following Amazon Web Services (AWS) instance types are supported with {produ
...
+|
+|x
+|x
+
+|`m6g.16xlarge`
+|
+|x
+|x
+
+|`c6g.large`
```
Comment (yunjiang29) [inline modules/installation-supported-aws-machine-types.adoc:411]: AFAIK, `m6g.*` machine types had been tested by QE, not sure if `c6g.*` had been tested, please @shivanthi-amara help to confirm, thanks.

Comment (shivanthi-amara) [reply modules/installation-supported-aws-machine-types.adoc:411]: @yunjiang29 - Yes, I tested with a  c6g.2xlarge cluster and also ran an E2E test. Here's the Jenkins job - https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/parallel-runner/7680/parameters/

The pass percentage is 91 %

Comment (yunjiang29) [reply modules/installation-supported-aws-machine-types.adoc:411]: @shivanthi-amara thanks for confirming. 
Are there any machine-type-related bugs? If not, I think we can add `c6g.*` to the list.

File: modules/installation-supported-aws-machine-types.adoc (hunk 1 above)
Comment (SaravanaStorageNetwork) [reply modules/installation-supported-aws-machine-types.adoc:16]: +1 to @kmccarron-rh

**Comment by jeffdyoung** [comment]
👍 from devel. 
We're good with what QE can test, at some point we may need to add other machine types:
https://aws.amazon.com/ec2/instance-types/
```
- T4g
- A1
- C6gn
- C7g (Graviton3, not widely available yet?)
- R6g
- X2gd
- G5g
- lm4gn
- ls4gen
```

File: modules/installation-supported-aws-machine-types.adoc (hunk 1 above)
Comment (Prashanth684) [reply modules/installation-supported-aws-machine-types.adoc:16]: maybe "machines based on x86_64 architecture" ? also i'm not sure if the usual doc nomenclature is to refer to the arch by go's amd64 or x86_64 like the kernel.

File: modules/installation-supported-aws-machine-types.adoc (hunk 3 above)
Comment (shivanthi-amara) [reply modules/installation-supported-aws-machine-types.adoc:411]: @kelbrown20 - I confirmed with Yunfei and we are good to merge this PR

**Comment by kelbrown20** [comment]
Thank you for the feedback @Prashanth684 @SaravanaStorageNetwork! I just updated the PR to change the title to `Machine types based on x86_64 architecture` and I was wondering how it looked?

**Comment by Prashanth684** [comment]
> Thank you for the feedback @Prashanth684 @SaravanaStorageNetwork! I just updated the PR to change the title to `Machine types based on x86_64 architecture` and I was wondering how it looked?

looks good.thanks!

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**